src/implementations/twig/components/banner/banner.html.twig

```diff
@@ -23,15 +49,29 @@
 {% set _css_class = 'ecl-banner' %}
 {% set _extra_attributes = 'data-ecl-banner data-ecl-auto-init="Banner"' %}
 {% set _picture = picture|default({}) %}
-{% set _variant = variant|default('plain-background') %}
+{% set _video = video|default({}) %}
 {% set _size = size|default('m') %}
-{% set _centered = centered ?? false %}
+{% set _font_size = font_size|default('m') %}
+{% set _font_color = font_color|default('dark') %}
+{% set _background = background|default('light') %}
+{% set _horizontal = horizontal|default('left') %}
+{% set _vertical = vertical|default('center') %}
 {% set _full_width = full_width|default(false) %}
 {% set _title = title|default('') %}
 {% set _description = description|default('') %}
 {% set _image = image|default('') %}
 {% set _credit = credit|default('') %}
 {% set _link = link|default({}) %}
+{% set _sr_play = sr_play|default('') %}
+{% set _sr_pause = sr_pause|default('') %}
+{% set _icon_path = icon_path|default('') %}
+
+{# DEPRECATED #}
+{% set _variant = variant|default('') %}
```

Comment:
- confusing, why is the variant "deprecated"? We still set the corresponding selector, we still have styles associated to it, we still kind of show a story per variant..

- We say in the docs (readme e comment block) that plain-background is the default for this parameter but instead there is no default.

- In the demo it seems really odd to have "image" and "text-box" looking almost exactly the same, "image" seems a bugged implementation of "text-box" with a missing value for the variant, like this...

I don't know, i find the stories really confusing, in the end it seems we have:

- plain background
- text-box (with image or video)
- text-overlay (with image or video)

Why not showing these three stories and having a control to switch between image and video, then..?